skymp5-server/cpp/server_guest_lib/FileDatabase.cpp

```diff
@@ -24,7 +24,8 @@ size_t FileDatabase::Upsert(const std::vector<MpChangeForm>& changeForms)
   size_t nUpserted = 0;
 
   for (auto& changeForm : changeForms) {
-    auto filePath = p / changeForm.formDesc.ToString('_');
+    std::string fileName = changeForm.formDesc.ToString('_') + ".json";
+    auto filePath = p / fileName;
     std::ofstream f(filePath);
     if (f) {
       f << MpChangeForm::ToJson(changeForm).dump();
```

Comment:
I think this should work
```suggestion
      f << MpChangeForm::ToJson(changeForm).dump(2);
```